src/bls/bls.h

```diff
@@ -158,7 +163,7 @@ class CBLSWrapper
     template <typename Stream>
     inline void Serialize(Stream& s, const bool specificLegacyScheme) const
     {
-        s.write(AsBytes(Span{ToByteVector(specificLegacyScheme).data(), SerSize}));
+        s.write(AsBytes(Span{ToBytes(specificLegacyScheme).data(), SerSize}));
```

Comment:
here's UB: ToBytes returns temporary object and Span doesn't own it.

```suggestion
        const auto bytes{ToBytes(specificLegacyScheme)};
        s.write(AsBytes(Span{bytes.data(), SerSize}));
```